substrate/frame/src/lib.rs

```diff
@@ -528,7 +540,11 @@ pub mod derive {
 }
 
 pub mod hashing {
```

Comment:
you can name it `cryptography`, I don't think `hashing` makes sense with the new items

Reply:
done